.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+	presets:['es2015'] 
```

Comment:
Do you really need this? As I see you specify all necessary presets there: https://github.com/rise2semi/react-comments/pull/16/files#diff-b9e12334e9eafd8341a6107dd98510c9R81